### ISSUE TITLE
The absence of a variable in the file close.less

### ISSUE THREAD

**Issue by AlekVolsk** [uikit/uikit#2266]
uikit/src/less/components/close.less

add a variable to specify large icons close:
```
.uk-close-large::before { background-image: data-uri('image/svg+xml;charset=UTF-8', "../../../dist/images/close-large.svg"); }
```

replace this:
```
@internal-close-image-large:                     "../../../dist/images/close-large.svg";
.uk-close-large::before { background-image: data-uri('image/svg+xml;charset=UTF-8', @internal-close-image-large); }
```

**Comment by dre1080** [comment]
:+1:

**Comment by philoest** [comment]
Thanks for reporting has been fixed in ca92c85
